README.md

```diff
@@ -1,37 +1,111 @@
 # Dolby.io Virtual World plugin for Unreal Engine
-Plugin integrating [Dolby.io Communications](https://dolby.io) with the Unreal Engine.
+Plugin integrating [Dolby.io Communications APIs](https://dolby.io) with the Unreal Engine.
 
 ## Supported platforms
 The plugin is compatible with the following Unreal Engine versions:
 - 4.27.2
 - 5.0.3
 
-The plugin is compatible with the following operating systems:
-- Windows 10 or newer
-- macOS:
-    - UE4: latest Mojave or newer
-    - UE5: latest Monterey or newer
```

Comment:
Should this information have been removed?

Reply:
But now that information about exact versions of Unreal was removed someone may think we are supporting 4.0 or 5.1. For now the plugin is only built for 4.27 and 5.0.

Reply:
The supported UE versions are mentioned in prerequisites.